runtime/testy/runtime.py

```diff
@@ -126,25 +128,51 @@ def checkTimedOut(func, *args):
 
     if testName != "":
       testName += "_"
-    setupFunc = getattr(studentCode, testName + "setup")
-    mainFunc = getattr(studentCode, testName + "main")
+    try:
+      setupFunc = getattr(studentCode, testName + "setup")
+    except AttributeError:
+      raise RuntimeError("Student code failed to define '{}'".format(test_name + "setup"))
+    try:
+      mainFunc = getattr(studentCode, testName + "main")
+    except AttributeError:
+      raise RuntimeError("Student code failed to define '{}'".format(test_name + "main"))
+
+    ensure_is_function(testName + "setup", setupFunc)
+    ensure_is_function(testName + "main", mainFunc)
+    ensure_not_overridden(studentCode, 'Robot')
+    ensure_not_overridden(studentCode, 'delay')
```

Comment:
This line is no longer needed. Instead, you should ensure that `Actions` is not overridden (but that's once `Actions` is actually defined)